Fabric.Authorization.API/Modules/IdentitySearchMetadataModule.cs

```diff
@@ -44,6 +44,11 @@ public IdentitySearchMetadataModule(ISwaggerModelCatalog modelCatalog, ISwaggerT
                         Code = (int) HttpStatusCode.OK,
                         Message = "OK"
                     },
+                    new HttpResponseMetadata<IEnumerable<IdentitySearchResponse>>
+                    {
+                        Code = (int) Nancy.HttpStatusCode.PartialContent,
+                        Message = "Partial success (e.g., results were found in Fabric.Authorization but the call out to Fabric.Identity failed)"
```

Comment:
nice addition. do you think callers will understand what data we get from Identity vs. Authorization?

Reply:
@kpaul , good point. Maybe I should explicitly call out which pieces of data come from Identity here so it's in the documentation.